### ISSUE TITLE
When creating new ItemProtoSet there are two options.

### ISSUE THREAD

**Issue by LeeWannacott** [peter-kish/gloot#109]
![image](https://github.com/peter-kish/gloot/assets/49783296/fe3d4918-2016-4171-8da9-580e2a18c303)

When creating a new ItemProtoSet there are two options ItemProtoSet and ItemProtoset. I think they both work, however I think this might be confusing to users. Any chance can just have one option? or is there a rationale for two?

**Comment by peter-kish** [comment]
Yeah, I noticed that, but I'm unsure if there's anything I can do about it or if it is a Godot thing. The `ItemProtoSet` class is registered with:
```gdscript
add_custom_type("ItemProtoSet", "Resource", preload("core/item_protoset.gd"), preload("images/icon_item_protoset.svg"))
```
Other GLoot classes are registered the same way, but don't show up twice in the dialog.

**Comment by LeeWannacott** [comment]
@peter-kish  If you change: `class_name ItemProtoset` to `class_name ItemProtoSet` in item_protoset.gd. You end up with only one option for adding resource:

![image](https://github.com/peter-kish/gloot/assets/49783296/e070c5e7-e415-442a-b2ad-2de2acc39e9d)

Although this causes a bunch of other problems..., so it's only part of the solution.

**Comment by peter-kish** [comment]
You were right, it boiled down to a typo. Should be fixed in eda7ab403cf4b886b3b1f85d742bed1c20953d99 on the [dev_v2_3_3](https://github.com/peter-kish/gloot/tree/dev_v2_3_3) branch.
Will close the issue once it gets merged to master.

**Comment by peter-kish** [comment]
Changes have been merged to master
